src/fn/move-contacts.js

```diff
@@ -21,6 +22,29 @@ module.exports = {
   }
 };
 
+const moveReportsBatch = async (db, contactIds, skip, replacementLineage, contactId, options) => {
```

Comment:
Since `skip` is the parameter that changes, please put it last. 
Also, more generally, this function receives a lot of parameters. Could this be refactored to reduce the number of parameters and make this more readable? 
For example, `options` is super generic. It's not obvious what the options are for, unless I scan and see that it's actually for writing to disk.